### PR TITLE
fix: use correct url to spotify & fix netease

### PR DESCRIPTION
**Pull request by Lseoksee**
I think you missed it while editing #2859.

### CODE REVIEW

**Review by rxri — APPROVED** [review]
i did yeah

File: CustomApps/lyrics-plus/Providers.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const Providers = {
 		const id = info.uri.split(":")[2];
 		let body;
 		try {
-			body = await CosmosAsync.get(baseURL + id);
+			body = await Spicetify.CosmosAsync.get(baseURL + id);
```
Comment (Lseoksee) [inline CustomApps/lyrics-plus/Providers.js:16]: In addition, Spotify's official lyrics were not working, so I checked and it seemed to be missing, so I modified this as well, but this exception occurred. 

`Error: GET request to wg://lyrics/v1/track/20uUaRkfRJZG15mXfn7LaU request failed with error code -1 (Resolver not found!)
    at l.onFailure (xpui.js:1:2244314)
    at onFailure (xpui.js:1:2243916)`

Does CosmosAsync api not support Spotify official lyrics yet?

Comment (rxri) [reply CustomApps/lyrics-plus/Providers.js:16]: wg endpoint doesn't exist anymore

Comment (rxri) [reply CustomApps/lyrics-plus/Providers.js:16]: I pushed fix to your fork

**Comment by Lseoksee** [comment]
The same applied to popupLyrics.

**Review by afonsojramos — APPROVED** [review]

**Pull request merged**